libs/voicefocus/worklet-inline-node.d.ts

```diff
@@ -1,5 +1,6 @@
 import { ProcessorMessage, VoiceFocusAudioWorkletNode, VoiceFocusNodeOptions, WorkerMessage } from './types.js';
 declare class VoiceFocusInlineNode extends VoiceFocusAudioWorkletNode {
+    private delegate?;
```

Comment:
Where is the corresponding TypeScript change for this change?

Reply:
The `.d.ts` and `.js` files in this directory are compiled from the upstream source (CR-56162358 and 5ba157430cf0e339cfeda516be8a841cb12111dd for this change).

You can see the code that actually assigns to this in `worklet-inline-node.js`.

`libs/` is the usual place to put vendored Node modules, and modules are usually delivered in their compiled form.